src/elbv2.rs

```diff
@@ -0,0 +1,3 @@
+//! AWS Application Load Balancing
+
+include!(concat!(env!("OUT_DIR"), "/elbv2.rs"));
```

Comment:
A TODO for later: will the `ElbClient` naming collision between ELB and ALB cause issues?